### PR TITLE
[Neuralchat] Notebook retrieval update

### PR DESCRIPTION
**Pull request by Liangyx2**
Update Notebook about Retrieval document format support.

### CODE REVIEW

File: intel_extension_for_transformers/neural_chat/requirements_hpu.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ fschat==0.2.32
 git+https://github.com/EleutherAI/lm-evaluation-harness.git@cc9778fbe4fa1a709be2abed9deb6180fd40e7e2
 intel_extension_for_pytorch
 neural-compressor
+neural_speed
```
Comment (hshen14) [inline intel_extension_for_transformers/neural_chat/requirements_hpu.txt:9]: why hpu requires neural-speed?

Comment (Liangyx2) [reply intel_extension_for_transformers/neural_chat/requirements_hpu.txt:9]: When I verified the code, an error occured: image,after `pip install neural-speed`, it runs smootly.
image

Comment (hshen14) [reply intel_extension_for_transformers/neural_chat/requirements_hpu.txt:9]: neural speed has nothing related with hpu. so please root cause the issue.

Comment (Liangyx2) [reply intel_extension_for_transformers/neural_chat/requirements_hpu.txt]: Solved. neural speed is no longer imported on hpu now.

**Comment by VincyZhang** [comment]
@Liangyx2 @lvliang-intel ready for merge?

**Pull request merged**